### PR TITLE
Fetch download count on regular basis

### PR DESCRIPTION
**Pull request by jvanbaarsen**
In order to keep track of the total downloads we have in Intercity, we created a
bash script on the Intercity.io server that will write to a HTML file how many
downloads per day there have been.

Since those stats are log rotated, we need to fetch them on a regular basis,
that way we have historical data.

All stuff is openly visible for now, only the sidekiq web view is locked behind
a password.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -C config/puma.rb
```
Comment (michiels) [inline Procfile:1]: I think you can drop the `-C config/puma.rb` since that's already pma's default.

File: app/jobs/fetch_download_count_job.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class FetchDownloadCountJob < ApplicationJob
+  def perform
+    data = fetch_data
+    data.each_line do |line|
+      splitted_data = line.split(" ")
+      Download.set_download_count(splitted_data[0], splitted_data[1].to_i)
```
Comment (michiels) [inline app/jobs/fetch_download_count_job.rb]: Maybe change the modal name to DailyDownloads or something?

**Review by michiels — COMMENTED** [review]
I posted two little comments on naming style, otherwise good to go!

File: app/jobs/fetch_download_count_job.rb (hunk 2 above)
Comment (jvanbaarsen) [reply app/jobs/fetch_download_count_job.rb]: :+1:

File: Procfile (hunk 1 above)
Comment (jvanbaarsen) [reply Procfile:1]: @michiels It does not state that in the readme:

> If you're using a configuration file, use the preload_app! method, and be sure to specify your config file's location with the -C flag:

Comment (michiels) [reply Procfile:1]: Ok weird, puma has always used config/puma.rb for me :p But yeah if it says so in the readme, lets keep it!

**Pull request merged**